recipes/aeppl/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set version = "0.0.6" %}
+
+package:
+  name: aeppl
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/aeppl/aeppl-{{ version }}.tar.gz
+  sha256: b0162e21c86ace1c698d5104aac003c994528ce46cd50f643b5a86ed6852dc2d
+  patches:
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Without the compilers, this can be noarch Python:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  noarch: python
```